VeteranCenters/Veteran_Centers.py

```diff
@@ -0,0 +1,82 @@
+import os
+import sys
+from datetime import datetime
+import re
+import numpy as np
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+from pymongo import MongoClient, errors
+from tqdm import tqdm
+import logging
+from io import StringIO
+
+logger = logging.getLogger(__name__)
+
+_i = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+if _i not in sys.path:
+    # add parent directory to sys.path so utils module is accessible
+    sys.path.insert(0, _i)
+del _i  # clean up global name space
+from shared_code.utils import (
+    check_similarity, locate_potential_duplicate,
+    insert_services, client
+)
+from shared_code.base_scraper import BaseScraper
+
+class VeteranCentersScraper(BaseScraper):
+
+    def grab_data(self):
+        url = 'https://sandbox-api.va.gov/services/va_facilities/v0/facilities/all'
+        headers = {'apikey': '1oavZAY4gJeEPE6atbHdXNN8hvKrXXYX', 'accept': 'text/csv'}
```

Comment:
You shouldn't add the api key in code. use it using the same technique as DBUSERNAME and PW , I will add it to azure function environment so that it becomes available as env variable when scraper runs in cloud. Let's use the key name as `VETERANS-API-KEY` , so you should replace this code by `os.environ['VETERANS-API-KEY']`

Reply:
done!